app/tests/system/nightwatch-config.js

```diff
@@ -1,9 +1,10 @@
 var path = require('path');
 var appPath = path.join(process.cwd(), '../ios/build/Build/Products/Debug-iphonesimulator/scaffold.app');
+var apkPath = path.join(process.cwd(), '../android/scaffold/build/outputs/apk/scaffold-debug.apk');
 
 module.exports = {
     'src_folders': ['./tests/system'],
-    'output_folder': './reports',
+    'output_folder': '${CIRCLE_TEST_REPORTS}',
```

Comment:
Does this mean we can only run tests when on Circle? How do we run them locally?